deploy/helm/sumologic/templates/pvc-cleaner/serviceaccount.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ template "sumologic.metadata.name.pvc-cleaner.roles.serviceaccount" . }}
+  labels:
+    app: {{ template "sumologic.labels.app.pvc-cleaner.roles.serviceaccount" . }}
+    {{- include "sumologic.labels.common" . | nindent 4 }}
+  {{- with (dig "pvc-cleaner" "serviceAccount" "annotations" false (.Values | merge (dict))) }}
+  annotations:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+{{- if (dig "pvc-cleaner" "job" "pullSecrets" false (.Values | merge (dict))) }}
+imagePullSecrets:
+{{ toYaml (dig "pvc-cleaner" "job" "pullSecrets" false (.Values | merge (dict))) | indent 2 }}
+{{- end }}
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: {{ template "sumologic.metadata.name.pvc-cleaner.roles.serviceaccount" . }}
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - persistentvolumeclaims
+  verbs:
+  - get
+  - list
+  - delete
+- apiGroups:
+  - "autoscaling"
+  resources:
+  - horizontalpodautoscalers
+  verbs:
+  - get
+  - list
+- apiGroups:
+  - ""
+  resources:
+  - pods
+  verbs:
+  - get
+  - list
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: {{ template "sumologic.metadata.name.pvc-cleaner.roles.serviceaccount" . }}
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: {{ template "sumologic.metadata.name.pvc-cleaner.roles.serviceaccount" . }}
+subjects:
```

Comment:
This was based on internal setup. Strangely enough, this wasn't necessary when tested manually, but integration tests didn't work without this role.